addons/web/static/src/scss/layout_boxed.scss

```diff
@@ -1,7 +1,7 @@
 .o_boxed_footer, .o_boxed_header, .o_report_layout_boxed {
     font-family: 'Lato';
     color: gray('700');
-    font-size: 12px;
+    font-size: 15px;
```

Comment:
While we are at it, we should use points (pt) instead of pixels as we are defining font size for printing purpose. 15px should = 11pt